config/env/production.js

```diff
@@ -1,5 +1,20 @@
+"use strict";
+
 module.exports = {
-  env: 'production',
-  databaseUrl: process.env.MONGO_URL || 'mongodb://martinbedouret:Muni1909@ds253587.mlab.com:53587/cboard',
-  jwt: { secret: process.env.JWT_SECRET || 'secret key for testing' }
+    env: 'production',
+    databaseUrl: process.env.MONGO_URL || 'mongodb://localhost/cboard-api',
+    jwt: {
+        secret: process.env.JWT_SECRET || 'secret key for testing'
```

Comment:
We should remove `|| 'secret key for testing'` for the *production* config. If the `JWT_SECRET` has not been supplied in the env, we should fail to start. Otherwise, this "secret" is public & compromised.

Reply:
fixed!